### PR TITLE
Front end rework

### PR DESCRIPTION
**Pull request by JohananLM**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: app/backend/app_new.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+# Use the current user identity to authenticate with Azure OpenAI, Cognitive Search and Blob Storage (no secrets needed, 
+# just use 'az login' locally, and managed identity when deployed on Azure). If you need to use keys, use separate AzureKeyCredential instances with the 
+# keys for each service
+# If you encounter a blocking error during a DefaultAzureCredntial resolution, you can exclude the problematic credential by using a parameter (ex. exclude_shared_token_cache_credential=True)
+azure_credential = DefaultAzureCredential(exclude_shared_token_cache_credential = True)
+
+# Used by the OpenAI SDK
+openai.api_type = "azure"
+openai.api_base = f"https://{AZURE_OPENAI_SERVICE}.openai.azure.com"
+openai.api_version = "2023-05-15"
```
Comment (superpoussin22) [inline app/backend/app_new.py:44]: Why hardrdcoding the api_version ?

File: app/backend/approaches/readretrieveread.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class ReadRetrieveReadApproach(Approach):
     """
 
     template_prefix = \
-"You are an intelligent assistant helping Contoso Inc employees with their healthcare plan questions and employee handbook questions. " \
+"You are an AI assistant that helps people find informatio. " \
```
Comment (superpoussin22) [inline app/backend/approaches/readretrieveread.py:30]: little typo : information.

**Comment by pamelafox** [comment]
@JohananLM Is this intended to be a public PR?

**Comment by JohananLM** [comment]
> @JohananLM Is this intended to be a public PR?

Sorry, I intended to PR my fork. This is not meant to be a public PR.

**Pull request closed without merging**